### ISSUE TITLE
unindent line when inserting {{

### ISSUE THREAD

**Issue by bsylvain** [joukevandermaas/vim-ember-hbs#15]
How to reproduce :
set up this text
```
{{#if true}}
  text text
{{/if}}
```
exit insert mode. put your cursor on the text line anywhere after the first t, insert `{{` it will remove the indentation.
If there is multiple lines of text it will only occurs on the first line after the handlebar tag.

**Comment by housleyjk** [comment]
Yeah, this irritates me to no end, but I've found that it disappears if the filetype is set to just plain `handlebars` instead of `html.handlebars`. Using plain `ft=handlebars` plays better with other plugins I'm using, so it's been a complete fix for me.
